src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -29,7 +29,7 @@ public void AddTrace(string trace)
 
         internal LogLevel LogLevel { get; set; } = LogLevel.Info;
 
-        internal ConcurrentDictionary<string, int> UnfinishedFailedMessages = new ConcurrentDictionary<string, int>();
+        internal ConcurrentDictionary<string, bool> UnfinishedFailedMessages = new ConcurrentDictionary<string, bool>();
```

Comment:
Yes. @tmasternak and I went over it on friday and we are pretty sure it should work ;)